src/classes/mech/Mech.ts

```diff
@@ -615,6 +615,7 @@ class Mech implements IActor {
     if (stress > this.MaxStress) this._current_stress = this.MaxStress
     else if (stress < 0) this._current_stress = 0
     else this._current_stress = stress
+    if (this._current_stress === 0 ) this._pilot.State.ReactorCriticalDestruct()
```

Comment:
This has been brought in line with CurrentStructure handling to deal with losing the last point.